.gitignore

```diff
@@ -31,3 +31,5 @@
 yarn-debug.log*
 .yarn-integrity
 .vscode/
+
+package-lock.json
```

Comment:
i don't think this is wise. keeping this lock file in the git history ensures that everyone is working off of the same dependency versions